.github/workflows/stage.sh

```diff
@@ -16,5 +16,5 @@ make clean package
 for f in repository/*.tgz; do mv $f $(echo $f | sed s/${version}/${slug}/); done
 gsutil cp -a public-read repository/*.tgz gs://projectriff/charts/snapshots/
 
-# upload uncharts
-gsutil cp -a public-read uncharted/*.yaml gs://projectriff/charts/uncharted/snapshots/${slug}/
+# upload releases
```

Comment:
is there a reason to still upload the intermediate charts for istio and kafka? Lines 15-17

Reply:
Same question with the release script

Reply:
No, should remove that